scripts/collect_steps.py

```diff
@@ -16,15 +17,22 @@
     "--features=enable-benches debug-trace step-trace",
     "--",
     "--num-multi-bits",
-    "1",
+    "3",
 ]
 QUERY_SIZE = 10
 PER_USER_CAP = [1, 3]
 ATTRIBUTION_WINDOW = [0, 86400]
-BREAKDOWN_KEYS = [1, 33]
+BREAKDOWN_KEYS = [8, 33]
 SECURITY_MODEL = ["malicious", "semi-honest"]
 ROOT_STEP_PREFIX = "protocol/alloc::string::String::run-0"
-EXPECTED_HEADER_LINES_COUNT = 4
+
+DEPTH_DYNAMIC_STEPS = "ipa::protocol::attribution::InteractionPatternStep"
+BIT_DYNAMIC_STEPS = [
+    "ipa::protocol::attribution::aggregate_credit::Step::compute_equality_checks",
+    "ipa::protocol::attribution::aggregate_credit::Step::check_times_credit",
+]
+MAXIMUM_DEPTH = 32
+MAXIMUM_BIT_LENGTH = 32
```

Comment:
while it seems valid to only consider inputs with less than 2B rows, I thought we will and already are decomposing match keys of 40 bits (and looking to use 64 bit match keys in the future)

Reply:
Bit-dynamic steps are from fields which its bit-length are determined at runtime (e.g. breakdown keys). Match key's bit-length is fixed in the code so the sort protocol generates complete steps.